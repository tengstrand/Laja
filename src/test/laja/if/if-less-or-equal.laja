#write "target/test-output/if-less-or-equal.out"
  #set (x = 1)
  #if (x <= 1)
x is less than or equal to 1
  #end
  #if (x <= 0)
x is less than or equal to 0
  #else
x is not less than or equal to 0
  #end
#end