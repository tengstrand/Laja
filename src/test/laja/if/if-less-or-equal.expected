x is less than or equal to 1
x is not less than or equal to 0
